src/pal/inc/pal.h

```diff
@@ -2410,12 +2410,12 @@ TlsFree(
     IN DWORD dwTlsIndex);
 
 PALIMPORT
-void *
+PVOID
```

Comment:
Could you please change these to PVOID in the implementation as well?

Reply:
Changed to `PVOID` in cpp files. Also reverted from CRT functions.